playbooks/user-password.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Generate password for user
+# You can also generate a password with `mkpasswd -m sha-512`
+- hosts: all
+  become: true
+  gather_facts: false
+  vars:
+    pwgen: "{{ lookup('password', '/dev/null length=8 chars=ascii_letters') }}"
+  tasks:
+    - set_fact:
+        passwd: "{{ pwgen }}"
+
+- hosts: all
+  become: true
+  tasks:
+    - name: Configure SSH to allow login with password
+      include_role:
+        name: dev-sec.ssh-hardening
+      vars:
+        ssh_client_hardening: false
+        ssh_server_password_login: true
+        ssh_use_pam: true
+        ssh_max_auth_retries: 10
+    - name: Set user password
+      include_role:
+        name: users
+      vars:
+        # User Configuration
+        users:
+          - username: "{{ ansible_user }}"
```

Comment:
I thought we were trying to get rid of this?

Reply:
I want to get rid of SETTING it, it's ok to read it. I don't want to require people to change config files and set that variable. Ansible sets it to the SSH user so it's OK to read.